app/javascript/crayons/MarkdownToolbar/__tests__/MarkdownToolbar.test.jsx

```diff
@@ -8,6 +8,9 @@ describe('<MarkdownToolbar />', () => {
   beforeEach(() => {
     global.Runtime = {
       getOSKeyboardModifierKeyString: jest.fn(() => 'cmd'),
+      isNativeIOS: jest.fn(() => {
```

Comment:
This is out of scope of this PR, but we should formalize some mocks for certain things so they can be reused instead of copy pasta.

Reply:
Yeah I think that would be a good plan 👍